consensus/src/tests/consensus_tests.rs

```diff
@@ -263,12 +269,12 @@ async fn not_enough_support() {
     for _ in 1..=3 {
         let certificate = rx_output.recv().await.unwrap();
         assert_eq!(certificate.round(), 3);
-    }
+    }*/
     let certificate = rx_output.recv().await.unwrap();
-    assert_eq!(certificate.round(), 4);
+    assert_eq!(certificate.round(), 1);
 }
 
-// Run for 7 dag rounds. Node 0 (the leader of round 2) is missing for rounds 1 and 2,
+// Run for 5 dag rounds. Node 0 (the leader of round 2) is missing for rounds 1 and 2,
```

Comment:
Was testing the fallback commit code path. Since the fallback wave 1 consists of rounds 1-4, we run for 5 DAG rounds so that the certificate in round 5 triggers a fallback commit.